src/mongoc/mongoc-uri.c

```diff
@@ -990,6 +1046,15 @@ mongoc_uri_unescape (const char *escaped_string)
 }
 
 
+const mongoc_read_prefs_t *
+mongoc_uri_get_read_prefs_t (const mongoc_uri_t *uri) /* IN */
```

Comment:
Since it's an API addition, see: https://github.com/mongodb/mongo-c-driver/blob/master/CONTRIBUTING.md for where to add the symbol for export/documentation.

As for deprecation, see examples like:

```c
void mongoc_bulk_operation_delete_one  (mongoc_bulk_operation_t       *bulk,
                                        const bson_t                  *selector)
   BSON_GNUC_DEPRECATED_FOR (mongoc_bulk_operation_remove_one);
```

That will trigger a deprecation warning, mentioning mongoc_bulk_operation_remove_one, every time a consumer uses mongoc_bulk_operation_delete_one